src/views/portfolio/projects/ProjectDetailsModal.vue

```diff
@@ -174,6 +188,39 @@
         }).catch((error) => {
           this.$toastr.w(this.$t('condition.unsuccessful_action'));
         });
+      },
+      retrieveParents: function() {
+        let url = `${this.$api.BASE_URL}/${this.$api.URL_PROJECT}/withoutDescendantsOf/${this.$props.uuid}`;
+        this.axios.get(url).then((response) => {
+          for (let i = 0; i < response.data.length; i++) {
+            let project = response.data[i];
+            if (project.uuid !== this.project.uuid){
+              if (project.version){
+                this.availableParents.push({value: project.uuid, text: project.name + ' : ' + project.version});
+              } else {
+                this.availableParents.push({value: project.uuid, text: project.name});
+              }
+            }
+            if (this.project.parent && this.project.parent.uuid === project.uuid  ) {
+              this.selectedParent = project.uuid;
+            }
+          }
+        }).catch((error) => {
+          this.$toastr.w(this.$t('condition.unsuccessful_action'));
+        });
+      },
+      hasActiveChild: function (project) {
+        let bool = false;
+        if (project.children){
+          for (const child of project.children){
+            if (child.active || bool){
+              return true;
+            } else {
+              bool = this.hasActiveChild(child);
+            }
+          }
+        }
+        return bool;
```

Comment:
```suggestion
        return project.children && project.children.some((child) => {
          return child.active || this.hasActiveChild(child);
        });

        return false;
```
Try this to simplify, also I think your logic is flawed in some situations